ddp-workspace/projects/ddp-rgp/src/app/app.module.ts

```diff
@@ -72,6 +71,7 @@ toolkitConfig.errorUrl = 'error';
 toolkitConfig.phone = '617-714-7395';
 toolkitConfig.infoEmail = 'raregenomes@broadinstitute.org';
 toolkitConfig.facebookGroupId = 'RareGenomesProject';
+toolkitConfig.mailingListDialogUrl = 'updates';
 
```

Comment:
Not one of your changes, but getting tslint warning on list 78. Suggest following change borrowed from ddp-prion:
```typescript
declare const ga: (...args: any[]) => void;
```

Reply:
I'll do that!  Thanks for the review